runway/data_types.py

```diff
@@ -405,27 +407,37 @@ class file(object):
         import runway
         from runway.data_types import file, category
 
-        inputs = {"folder": file(is_folder=True)}
+        inputs = {"directory": file(is_directory=True)}
         outputs = {"result": category(choices=["success", "failure"])}
         @runway.command("batch_process", inputs=inputs, outputs=outputs)
         def batch_process(result_of_setup, args):
-            result = do_something_with(args["folder"])
+            result = do_something_with(args["directory"])
             return { "result": "success" if result else "failure" }
 
     :param name: The name of this variable, defaults to None
     :type name: string, optional
-    :param is_folder: Does this variable represent a folder instead of a file? Defaults to False.
-    :type is_folder: bool, optional
+    :param is_directory: Does this variable represent a directory instead of a file? Defaults to False.
+    :type is_directory: bool, optional
     """
```

Comment:
We should add the docstring `:param` and `:type` for the new `suffix` parameter.